src/SFA.DAS.ApplyService.InternalApi/Services/AssessorLookupService.cs

```diff
@@ -233,6 +233,45 @@ public string GetTitleForPage(string pageId)
             }
         }
 
+        public string GetSectorNameForPage(string pageId)
+        {
+            switch (pageId)
+            {
+                case RoatpWorkflowPageIds.DeliveringApprenticeshipTraining.AgricultureEnvironmentalAndAnimalCare.MostExperiencedEmployee:
+                    return RoatpWorkflowPageIds.DeliveringApprenticeshipTraining.AgricultureEnvironmentalAndAnimalCare.Name;
```

Comment:
Could this be a dictionary, with the page ID as a key?

Reply:
I tried this and it's clunky (would need exception handling to deal with code flow) when a pageId is entered that doesn't exist - (this is possible through the API call), so I'm leaving as is.